e2e/02-login-flow.e2e.spec.ts

```diff
@@ -107,6 +107,7 @@ describe("Login Flow", () => {
   })
 
   it("navigates back to move home screen", async () => {
+    await scrollDown()
```

Comment:
I don't know what the issue is, but doing a `scrollDown()` to next do a `scrollUp()` feels like an anti-pattern to me

Reply:
if this works merely because this is synonym to a `wait sleep()`, we should use better primitive